src/source/Ice/ConnectionListener.c

```diff
@@ -216,6 +216,17 @@ STATUS connectionListenerStart(PConnectionListener pConnectionListener)
     return retStatus;
 }
 
+static BOOL canReadFd(INT32 fd, struct pollfd* fds, INT32 nfds)
```

Comment:
I would remove static scope. The codebase assumes Ansi C

Reply:
ok